hyperspy/component.py

```diff
@@ -119,8 +120,50 @@ def __init__(self):
         self.component = None
         self.grad = None
         self.name = ''
+        self.units = ''
```

Comment:
And actually we do, we just did not initialize it, as it is a `traits.Str("")`. This line is essentially only for clarity of the coder - that the default units are empty